modules/ROOT/pages/configure-multiple-credential-providers.adoc

```diff
@@ -0,0 +1,116 @@
+
+
+== The Use Case
+
+To better understand how to implement the multiple external credential provider feature, let us look at the example of "Good Weather" company that is in the business of forecasting weather for a given location and time. Good Weather collects quantitative data about the current state of the atmosphere at a given place and, using meteorology, predicts how that current state will change over time.
+
+Good Weather has several sub-organizations running under the master organization. These sub-organizations exist based on the functions they perform, for example, Engineering and Forecasting. The Forecasting department researches and collects recent meterological data from various sources. This data is then consumed by various client applications. While the Engineering department maintains the applications that host the forecasting research data.
+
+Good Weather needs to ensure that the forecasting data is accurate at all times. For that reason the weather APIs must be accessed only by employees of the Forecasting sub-organization. Good Weather also wants to ensure that the production APIs are the single source of truth at all times.
+
+To ensure security and segregation of access for these different organizations, environments, and APIs, Good Weather IT sets up a separate external client providers for the Forecasting sub-organization.
+
+//example of how they implement the client providers.
+//Tomas how will they best implement this?
+
+== Guidelines for Implementing Multiple Client Providers
+
+Before you implement multiple client providers in Anypoint Platform, ensure that you carefully review the following guidelines and best practices:
+
+* When changing the client provider for an existing API instance, ensure that the API:
+** Has no contracts
+** Has no client provider-related policies applied, such as PingFederate and OpenID policies.
+* When promoting and importing an API:
+
+* If you assign an external client provider to an environment, all existing APIs in that environment display Anypoint Platform in the UI by default. 
++
+You must then reassign the appropriate external client provider for that API.
+* After you assign an external client provider to an API, you cannot reassign the client provider back to the default native Anypoint Platform credential provider.
+* If you remove a client provider from an environment, all existing APIs and client applications using that client provider continue to work.
+* If you delete an client provider from a master organization, all existing APIs and client applications using that client provider default to the native Anypoint Platform client provider. 
++
+All contracts remain intact.
+
+
+== Configuring Multiple Client Providers User Flow
+
+Based on the use case explained in the previous section, Good Weather uses the following flow to implement multiple external client providers:
+
+image::multiple-IdP-user-flow.png[User Flow for Configuring Multiple External Client Providers]
+
```

Comment:
Should the graphic be followed by some text that sets the audience expectation for the following subtopics?

Reply:
Yes, those are the guidelines.

Reply:
Done! thanks!